### DISCUSSION TITLE
Is leaderboards going to be merged with stable?

### DISCUSSION THREAD

**Discussion by xanning** [ppy/osu#26814] [Q&A] [answered]
title.
Is it just a temporary thing, or stable scores will be converted into lazer and lazer leaderboards will be in same with Stable?
So no seperate leaderboards?
Also asking because all leaderboard, top plays etc disappeared.
(Watched last youtube video but wanted to be sure)

**Comment by peppy** [comment, marked as answer]
what you see if what you get for now. choose your mode on the site, stable or lazer.
